packages/flutter/lib/src/widgets/shortcuts.dart

```diff
@@ -902,22 +902,16 @@ class ShortcutManager with Diagnosticable, ChangeNotifier {
   /// [Action] must be enabled.
   @protected
   KeyEventResult handleKeypress(BuildContext context, KeyEvent event) {
-    final Intent? matchedIntent = _find(event, HardwareKeyboard.instance);
-    if (matchedIntent != null) {
-      final BuildContext? primaryContext = primaryFocus?.context;
-      if (primaryContext != null) {
-        final Action<Intent>? action = Actions.maybeFind<Intent>(
-          primaryContext,
-          intent: matchedIntent,
-        );
-        if (action != null) {
-          final (bool enabled, Object? invokeResult) = Actions.of(primaryContext).invokeActionIfEnabled(
-            action, matchedIntent, primaryContext,
-          );
-          if (enabled) {
-            return action.toKeyEventResult(matchedIntent, invokeResult);
-          }
-        }
+    late final Intent? intent = _find(event, HardwareKeyboard.instance);
```

Comment:
```suggestion
    // Marking some variables as "late" ensures that they aren't evaluated unless needed.
    final Intent? intent = _find(event, HardwareKeyboard.instance);
```

Reply:
That's a good point—performance-wise, `intent` doesn't need to be marked as `late`.

But I thought it'd still be nice to have it here, since the style guide recommends [aligning expressions](https://github.com/flutter/flutter/blob/master/docs/contributing/Style-guide-for-Flutter-repo.md#align-expressions).

I'll go ahead and add that comment, thanks!